Competitive Programming/c2cpp.md

```diff
@@ -0,0 +1,117 @@
+# C to Cpp for competitive
+
+This article is written for specifically keeping competetive in mind and not for using cpp in development or any other context. Also this aims at getting started to use good and important features of c++ as quickly as possible so skips many details.
+Most of the c++ fetures talked here work on c++98 and after but some work only after c++11 so for simplicity we will assume to be using c++11. Also we will be using gcc/g++ compiler. You can also use an IDE like codeblocks it already has gcc and g++ compilers by default (MINGW); remember to enable c++11 or c++14 option in codebloks' settings.
+
+Once you are familiar with C its easy to migrate to and learn C++. First of all since c++ is superset of C so you can use most of the concepts of c in c++. So we will first talk about porting code and then a couple of things new in c++ useful for competetive.
+
+# PREREQUISITES
```

Comment:
Use title-case, not full upper-case.

Reply:
I have been using the convention of adding an empty line after every heading. I'd like it if you do the same to maintain consistency throughout this repository.